@here/harp-mapview/lib/DecodedTileHelpers.ts

```diff
@@ -835,10 +835,21 @@ function getBaseColorPropName(technique: Technique): string | undefined {
     return getTechniqueDescriptor(technique)?.attrTransparencyColor;
 }
 
+type TypedArray =
+    | Int8Array
+    | Uint8Array
+    | Uint8ClampedArray
+    | Int16Array
+    | Uint16Array
+    | Int32Array
+    | Uint32Array
+    | Float32Array
+    | Float64Array;
+
 function getTextureBuffer(
     buffer: ArrayBuffer,
     textureDataType: THREE.TextureDataType | undefined
-): THREE.TypedArray {
+): TypedArray {
```

Comment:
I think this should be `BufferSource`, at least the `DataTexture` takes a BufferSource, see: https://github.com/three-types/three-ts-types/commit/4ef80d3e9e76a44271e21d8f536b784213375dbe